component/kibana-host.libsonnet

```diff
@@ -0,0 +1,22 @@
+local common = import 'common.libsonnet';
+local kap = import 'lib/kapitan.libjsonnet';
+local kube = import 'lib/kube.libjsonnet';
+local resourceLocker = import 'lib/resource-locker.libjsonnet';
+local inv = kap.inventory();
+// The hiera parameters for the component
+local params = inv.parameters.openshift4_logging;
+
+local routeToPatch = kube._Object('route.openshift.io/v1', 'Route', 'kibana') {
+  metadata+: {
+    namespace: params.namespace,
+  },
+};
+
+{
+  [if params.kibana_host != null then '32_kibana_host']:
+    resourceLocker.Patch(routeToPatch, {
+      spec: {
+        host: params.kibana_host,
```

Comment:
Did you check that the console link `kibana-public-url` is updated automatically (because I suspect it is not.. :yak:)

Reply:
It is surprisingly 🙃

Reply:
Ah nevermind you said it did. I'm surprised :sweat_smile: